cmd/device_info_linux.go

```diff
@@ -38,3 +39,29 @@ func getMac(iface string) (string, error) {
 	}
 	return strings.TrimSpace(string(macForIface)), nil
 }
+
+func getOSVersion() (string, error) {
+	osInfoPath := "/etc/os-release"
+
+	osFile, err := os.ReadFile(osInfoPath)
+	if err != nil {
+		return "", fmt.Errorf("cannot read file %w", err)
+	}
+
+	osInfo := make(map[string]string)
+	scanner := bufio.NewScanner(strings.NewReader(string(osFile)))
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		if key, value, found := strings.Cut(line, "="); found {
+			osInfo[key] = strings.Trim(value, `"`)
```

Comment:
strings.Trim will handle valid files fine, but could do unexpected things in the presence of unbalanced quotes or multiple quotes. To be safe, perhaps use a regex to remove a single pair of surrounding double quotes and leave everything else alone.